treeherder/webapp/api/performance_data.py

```diff
@@ -889,6 +906,46 @@ def _get_signatures(self, repository_name, framework, interval, no_subtests):
         signatures = self._get_signatures_values(signatures)
         return signatures
 
+    def _create_graph_links(
+        self,
+        base_repo_name,
+        new_repo_name,
+        base_revision,
+        new_revision,
+        framework,
+        timeRange,
+        signature,
+    ):
+        highlighted_revision_key = 'highlightedRevisions'
+        time_range_key = 'timeRange'
+        seriesKey = 'series'
+
```

Comment:
if not timerange:
            # The graph should display data for the last 7 days, unless mentioned otherwise
            timerange = 604800
```suggestion
This check is needed because when a base revision and a new revision are selected the timerange is set by default in the current logic for the graph link.

```